airflow/config_templates/config.yml

```diff
@@ -1529,6 +1529,19 @@ webserver:
       type: boolean
       example: ~
       default: "False"
+    trigger_dag_url:
```

Comment:
Why do we need this btw? (As in when would someone ever set it?)

I'm wary about hard-coding URLs like this in the  config, as this would need extra config if someone sets the webserver root to something other than `/` -- i.e. why can't we use `{{ url_for(..., dag_id=...) }}` in the template>

Reply:
This config entry belongs to part 2 of the AIP-50. As described in the PR header I implemented as proposal in a reduced manner to prevent an impact on DAG loading. The actual described feature in AIP-50 would require to load serialized DAG information into the overview page or an DB / model extension for actually a small feature and might have a performance impact w/o proper planning.

I also wanted to use `url_for()` but this is only a possibility for internal routing. The idea in AIP-50 was that you can define a path to a plugin or other custom web front-end to re-direct the user to instead of the internal view.

If you think this is "not good" I can revert this part and separate this out to another PR, I did not want to raise a discussion on this but wanted to close the gap to AIP-50 plans.

Reply:
Yeah, if you could separate this that would be good.

(One option for when re-doing it _might_ be store the route name in config, i.e. the value of the first argument to `url_for`?)

Reply:
Reverted code for part 2, let's discuss separately. From Falsk documentation I did not understand how to use `url_for()` for external links but maybe I need to re-read.